frontend/src/components/UI/TabPanels/Tabs.vue

```diff
@@ -116,12 +116,32 @@
 								<q-input
 									class="q-gutter-md"
 									filled
-									label="Repo *"
+									label="Git provider URL"
 									lazy-rules
 									:rules="[
```

Comment:
Remove the rules for: `GitProviderURL`, `GitOrganization` and `GitPersonalAccessToken` because is not mandatory :)